### ISSUE TITLE
TestAssertCommand always passes

### ISSUE THREAD

**Issue by cscetbon** [kudobuilder/kuttl#278]
**What happened**:
when the command exits with an error, the test still passes

**What you expected to happen**:
I would expect the test to fail as the command returns a non-zero code

**How to reproduce it (as minimally and precisely as possible)**:
See https://gist.github.com/cscetbon/4a597925abfd6ae353a0d97fc9b0b9c3 and here is my folder structure

```
tree .                                                                                                               
.
├── kuttl-test.yaml
└── testassertcommand
    └── 01-assert.yaml
```
**Environment**:
- Kubernetes version (use `kubectl version`): 1.18
- KUTTL version (use `kubectl kuttl version`): 0.9

**Comment by alenkacz** [comment]
okay this took me a while as well, the problem is you have uppercase Commands but it should be lowercase 🤦‍♀️ we should add some validation

**Comment by cscetbon** [comment]
Thank you @alenkacz, I’ll give it a shot. I’m surprised I can create a testassertcommand with actually not even one command/script as I’m using the wrong syntax and it gets ignored 🤦‍♂️
Also some documentation is missing

**Comment by morningspace** [comment]
@alenkacz @cscetbon Even I correct the TestAssert using the lowercased string instead of the capitalized string. I still it passed even it returns non-zero code (-1). Any idea?

```yaml
apiVersion: kuttl.dev/v1beta1
kind: TestAssert
commands:
- command: exit -1
```

**Comment by morningspace** [comment]
BTW: It looks I'm using 0.8.0. Is it because my KUTTLE is a bit old? I installed it from krew.
```console
kubectl kuttl -v
kubectl-kuttl version 0.8.0
```

**Comment by morningspace** [comment]
Wait, it looks this should be a feature added in release 0.8.0: https://github.com/kudobuilder/kuttl/pull/245. Did I miss anything?

**Comment by morningspace** [comment]
Alright, looks like it's just a proposal (KEP) and now implemented in 0.9.0, https://github.com/kudobuilder/kuttl/pull/272, assuming I need to install 0.9.0 instead.
